mne/preprocessing/_fine_cal.py

```diff
@@ -388,4 +390,14 @@ def write_fine_calibration(fname, calibration):
     """
     _check_fname(fname, overwrite=True)
     check_fname(fname, 'cal', ('.dat',))
-    raise NotImplementedError
+
+    # TODO: Figure out if write or write+binary mode is best
```

Comment:
use `wb` and e.g. `cal_file.write((cal_str + '\n').encode('ASCII'))`, it's clear you're writing an ASCII-encoded file that way